commons/src/types/Events.ts

```diff
@@ -19,8 +19,15 @@ interface _Event {
   locationString?: string;
   attendeeStatus?: "yes" | "no" | "noreply" | "maybe";
   isNewEvent?: boolean;
+  conferencing?: EventConferencing;
 }
 
+export interface EventConferencing {
+  provider?: "WebEx" | "Zoom Meeting" | "GoToMeeting" | "Google Meet";
+  details: {
+    url: string;
+  };
+}
```

Comment:
Added the basics of our conferencing object, per https://developer.nylas.com/docs/api/#post/events